src/commands/admin/issue.ts

```diff
@@ -1,83 +1,68 @@
-		const newIssue = await msg.client.octokit.issues.create({
+		const labels = label ? label.split(', ') : [];
+
+		const newIssue = await interaction.client.octokit.issues.create({
 			owner: 'ud-cis-discord',
-			repo: project,
+			repo: GITHUB_PROJECT,
 			title: title,
-			milestone: milestone || undefined,
 			labels: labels,
-			body: `${body}\n\n<sub>Created by ${msg.member.displayName} via ${BOT.NAME}</sub>`
+			body: body || `\n\n<sub>Created by ${interaction.user.username} via ${BOT.NAME}</sub>`
 		}).catch(response => {
 			console.log(response);
 			let errormsg = '';
 			const { errors } = response as RequestError;
 			errors.forEach(error => {
 				errormsg += `Value ${error.code} for field ${error.field}.\n`;
 			});
-			msg.channel.send(`Issue creation failed. (HTTP Error ${response.status})
+			interaction.channel.send(`Issue creation failed. (HTTP Error ${response.status})
 	\`\`\`diff
 	-${errormsg}\`\`\``);
 		});
-
 		if (newIssue) {
-			return msg.channel.send(`I've created your issue at <${newIssue.data.html_url}>`);
+			return interaction.reply(`I've created your issue at <${newIssue.data.html_url}>`);
 		} else {
-			return msg.channel.send('Something went horribly wrong with issue creation! Blame Josh.');
+			return interaction.reply('Something went horribly wrong with issue creation! Blame Josh.');
 		}
 	}
 
-	async argParser(_msg: Message, input: string): Promise<Array<string | string[]>> {
-		const [title, ...args] = input.split('--');
-		if (!title) throw `Usage: ${this.usage}`;
-
-		const splitArgs = args.map(arg => arg.trim());
-
-		let project = GITHUB_PROJECT;
-		let labels = [];
-		let milestone = '';
-		let body = '';
-
-		if (splitArgs.find(str => str.includes('project'))) {
-			[, project] = splitArgs.find(str => str.includes('project')).split('=');
-			if (!project) throw `Invalid format for project, more help:\n${this.extendedHelp}`;
-		}
-
-		if (splitArgs.find(str => str.includes('labels'))) {
-			labels = splitArgs.find(str => str.includes('labels'))
-				.split('=').slice(1).join().split(',').map(label => label.trim());
-			if (!labels) throw `Invalid format for labels, more help:\n${this.extendedHelp}`;
-		}
-
-		if (splitArgs.find(str => str.includes('milestone'))) {
-			[, milestone] = splitArgs.find(str => str.includes('milestone')).split('=');
-			if (!milestone) throw `Invalid format for milestone name, more help:\n${this.extendedHelp}`;
-		}
-
-		if (splitArgs.find(str => str.includes('body'))) {
-			[, body] = splitArgs.find(str => str.includes('body')).split('=');
-			if (!body) throw `Invalid format for body, more help:\n${this.extendedHelp}`;
-		} else if (splitArgs.find(str => str.includes('description'))) {
-			[, body] = splitArgs.find(str => str.includes('description')).split('=');
-			if (!body) throw `Invalid format for description, more help:\n${this.extendedHelp}`;
-		}
-		return [title, project, labels, milestone, body];
+	async run(msg: Message): Promise<Message | void> {
```

Comment:
make sure this returns nothing before we integrate the whole slashCommands branch into main but I don't know if that matters at the mo